components/Footer.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <footer class="usa-footer usa-footer--slim">
       <div class="grid-container usa-footer__return-to-top">
-        <a href="#">Return to top</a>
+        <nuxt-link :to="{ path: getUrl(), hash: '#' }" href="~#">Return to top</nuxt-link>
```

Comment:
Not sure we need that href attr. I think it's a little odd to not link to a particular ID (`#top` or something similar).

Reply:
https://stackoverflow.com/questions/4855168/what-is-href-and-why-is-it-used

Reply:
Interesting. I still think it's odd not to link to an anchor, and I haven't seen any examples of nuxt-link that use the href attribute. I see examples where folks have implemented a scrollTo behavior to get smooth scrolling to anchors... https://dev.to/dimer191996/nuxt-js-smooth-scrolling-with-hash-links-94a
https://router.vuejs.org/guide/advanced/scroll-behavior.html#async-scrolling